{"name": "Негативная проверка авторизации юзера с неправильным майлом или паролем", "status": "passed", "description": "Запрос должен вернуть ошибку и правильный код ответа", "attachments": [{"name": "stdout", "source": "0c6162dd-2662-403e-bdff-4023c7acc65e-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "wrong_value", "value": "'email'"}], "start": 1723366548771, "stop": 1723366549331, "uuid": "e5db4de8-f410-460d-b3cf-f8429cf369f2", "historyId": "56aba404929ab9e5ddcaaa166b8e2521", "testCaseId": "4d86218e2daabd1383135081cc8a8b93", "fullName": "tests.test_authorization_courier.test_authorization_with_wrong_password_or_email.TestAutorizationUser#test_authorization_user_with_wrong_password_or_email_error", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('reg')"}, {"name": "parentSuite", "value": "tests.test_authorization_courier"}, {"name": "suite", "value": "test_authorization_with_wrong_password_or_email"}, {"name": "subSuite", "value": "TestAutorizationUser"}, {"name": "host", "value": "DESKTOP-IKUPUFD"}, {"name": "thread", "value": "16804-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_authorization_courier.test_authorization_with_wrong_password_or_email"}]}